### ISSUE TITLE
Fix python.flask.security "injection" rules

### ISSUE THREAD

**Issue by minusworld** [returntocorp/sgrep-rules#954]
**Describe the bug**
The Python Flask injection rules are not working as expected; See this fixed version of `python.flask.security.os-system-injection`:

https://semgrep.dev/s/8xe7/?stepByStep=1

whereas this, the current version of the rule, is broken:

https://semgrep.dev/s/gpzn/?stepByStep=1

**Additional Context**
This could actually be a `semgrep` bug. Further investigation required.

**Comment by bored-engineer** [comment]
I'm not sure if this is the same issue, but I expected this to trigger but it does not, seems like the string concatenation is where it gets confused (removing `"echo" + ` detects properly):
```py
from flask import request

@app.route('/')
def index():
    command = request.args.get('command', 'whoami')
    os.system("echo " + command)
```
